foss4gna_24/.ipynb_checkpoints/big-EO-viz-checkpoint.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>to be clear the memory constraint reason for dropping data frames in the notebook sharing site</p><p>the dropping columns is both the notebook sharing site and the lonboard memory footprint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Thanks. Updated as follows-</p><p> </p><blockquote>In order to save memory, we delete the unrequired data frames to keep the memory in check for notebook sharing to be discussed in following section. Also, we drop the columns which are not relevant for the visualization to reduce the memory footprint for lonboard and notebook sharing.&nbsp;</blockquote>